facia-purger/src/main/scala/com/gu/purge/facia/ParseS3Path.scala

```diff
@@ -0,0 +1,20 @@
+package com.gu.purge.facia
+
+import org.parboiled2.{ Parser, ParserInput, Rule1 }
+
+import scala.util.Success
+
+// /aws-frontend-store/CODE/frontsapi/pressed/live/au/fapi/pressed.json
+class ParseS3Path(stage: String, val input: ParserInput) extends Parser {
+  def prefix = rule { s"$stage/frontsapi/pressed/live/" }
```

Comment:
I know we are not doing it religiously in frontend but I try to force myself to explicitly define the type of every val or def which is not obvious. It is not for the compiler which is smart enough but for my all my fellow devs who are reading the code and do not necessarily have IntelliJ (ctrl+p) around.
_By obvious I mean unless you invoke a constructor or something like a scalar types (int, double, ...) or string_